llvm/lib/Analysis/Delinearization.cpp

```diff
@@ -714,7 +895,140 @@ bool llvm::tryDelinearizeFixedSizeImpl(
   if (!SrcGEP)
     return false;
 
-  getIndexExpressionsFromGEP(*SE, SrcGEP, Subscripts, Sizes);
+  // When flag useGEPToDelinearize is false, delinearize only using array_info.
```

Comment:
`array_info` probably refers to another patch, it is not currently that appears in LLVM-IR.